File: javaapp/java-api/src/main/java/com/dreamteam/api/model/http/car/Brand.java
package com.dreamteam.api.model.http.car;

import com.dreamteam.api.model.http.HttpModel;
import lombok.Data;

import javax.validation.constraints.NotNull;

/**
 * Created by JK on 2017-10-11.
 */
@Data
public class Brand extends HttpModel {

    @NotNull(message = "Brand has to have name")
    private String name;

    public Brand() {
        super();
    }
}

File: Tas/src/app/models/brand.ts
export class Brand {
  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  constructor(id: number, name: string) {
    this._id = id;
    this._name = name;
  }
  _id: number;
  _name: string;
}

